modules/menus/dashboard/stats/index.ts

```diff
@@ -23,6 +22,11 @@ interval.connect('changed', () => {
     storageService.updateTimer(interval.value);
 });
 
+const handleClick = (): void => {
+    App.closeWindow('dashboardmenu');
+    Utils.execAsync(`bash -c "$TERMINAL -e btop"`).catch((err) => `Failed to open btop: ${err}`);
```

Comment:
```suggestion
    Utils.execAsync(`bash -c "${terminal} -e btop"`).catch((err) => `Failed to open btop: ${err}`);
```

Reply:
We can leave these options in but have the default terminal configured in the options to '$TERMINAL'. That way if the user doesn't have a default terminal set, and the click doing nothing, they can instead configure it here to change that.